tracer/src/Datadog.Trace/ITracer.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="ITracer.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace
+{
+    /// <summary>
+    /// The tracer is responsible for creating spans and flushing them to the Datadog agent
+    /// </summary>
+    public interface ITracer
+    {
+        /// <summary>
+        /// Gets the active scope
+        /// </summary>
+        IScope ActiveScope { get; }
+
+        /// <summary>
+        /// Gets the default service name for traces where a service name is not specified.
+        /// </summary>
+        string DefaultServiceName { get; }
```

Comment:
Hmm. I think it makes sense to remove it, especially because the service name can be retrieved through the Settings object. Not to mention, it's easier to add API's if users want it than to remove API's. I've proactively removed it in 00c432bfac48a9caafc5bf8d643060edb8837607 but let me know if you disagree and we can make changes

Reply:
> especially because the service name can be retrieved through the Settings object

Only if a custom `DD_SERVICE` is provided, otherwise the default service name is calculated in the `TracerManagerFactory`:

https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/TracerManagerFactory.cs#L60-L62

I'm not against removing it though. As you say, it's easy to add it back if we have any complaints